kernel/src/main/java/org/kframework/parser/inner/disambiguation/PriorityVisitor.java

```diff
@@ -77,11 +113,10 @@ public PriorityVisitor2(TermCons parent, Side side, POSet<Tag> priorities, Set<T
         }
 
         public Either<java.util.Set<KEMException>, Term> apply(TermCons tc) {
-            if (tc.production().att().contains(Att.BRACKET())) return Right.apply(tc);
             //if (Side.RIGHT  == side && !(tc.production().items().apply(0) instanceof NonTerminal)) return Right.apply(tc);
             //if (Side.LEFT == side && !(tc.production().items().apply(tc.production().items().size() - 1) instanceof NonTerminal)) return Right.apply(tc);
```

Comment:
Since we're open to changing the behavior of priorities, can we add these now? We could finally fix some long standing limitations.

Reply:
I'm still not persuaded we won't introduce new ambiguities that are difficult to solve into existing definitions if we do this. It's a much more intrusive change than what I'm suggesting. Also, unless I'm mistaken, if we were to naively uncomment these lines, the test I wrote would fail.